Readme.txt

```diff
@@ -1,41 +1,34 @@
-                              MORSE RUNNER  1.71
+                                MORSE RUNNER
                               Contest Simulator
-
                                   freeware
+                                  
+                          Field Day Contest Prototype
 
-               Copyright (C) 2004-2016 Alex Shovkoplyas, VE3NEA
-
-                      http://www.dxatlas.com/MorseRunner/
-
-
-
+                          Ver 1.80 Copyright (C) 2022  Mike Brashler, W7SST https://github.com/w7sst/MorseRunner
+Previous works:
+              original to Ver 1.68  Copyright (C) 2004-2016 Alex Shovkoplyas, VE3NEA http://www.dxatlas.com/MorseRunner
+              Ver 1.69 to Ver 1.71  Copyright (C) 2016 Lin Quan, BG4FQD https://github.com/BH1SCW/MorseRunner
+                          Ver 1.71a Copyright (C) 2021 David Palma, CT7AUP
 
```

Comment:
I wonder if we should add:
```
For more information on the Morse Runner Community Edition project, please visit https://github.com/w7sst/MorseRunner.
```
We could extend this statement with an invitation to join, perhaps... `Please consider joining our community effort`. Or something like that to pull the user into the project.

Or perhaps:
```
               Copyright (C) 2004-2016 Alex Shovkoplyas, VE3NEA
                      http://www.dxatlas.com/MorseRunner/

               Copyright (C) 2022 Contributors to the Morse Runner Community Edition
                      https://www.github.com/w7sst/MorseRunner/
```

Reply:
I like this!

Reply:
I also like this.

Reply:
Agreed. I'll add this.

Reply:
I added the following after the copyright info...
```
JOIN OUR COMMUNITY
  You are invited to join our community effort.
  For more information on the Morse Runner Community Edition project,
  please visit at https://github.com/w7sst/MorseRunner#readme.
```